app/controllers/api_controller/settings.rb

```diff
@@ -17,7 +17,7 @@ def show_settings
     private
 
     def exposed_settings
-      @exposed_settings ||= YAML.load_file(Rails.root.join("config/api_settings.yml"))[:settings]
+      Api::Settings.collections[:settings][:categories]
```

Comment:
Not currently, because `Api::Settings.collections` just returns a plain old hash. But it will once I've cleared the blockers for making it a `Config::Options`